manifests/h/HakuNeko/HakuNeko/Nightly/8.3.4/HakuNeko.HakuNeko.Nightly.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: HakuNeko.HakuNeko.Nightly
+PackageVersion: 8.3.4
+PackageLocale: en-US
+Publisher: Ronny Wegener
```

```suggestion
Publisher: The HakuNeko Development Team <https://github.com/orgs/manga-download/people>
```
This must match Control Panel.